pkg/rancher-components/src/components/Pill/RcStatusIndicator/RcStatusIndicator.vue

```diff
@@ -25,8 +25,8 @@ const props = defineProps<RcStatusIndicatorProps>();
         border: 1px solid transparent;
 
         &.disc {
-            width: 6px;
-            height: 6px;
```

Comment:
This was missed in the original review of the status-indicator. It's supposed to be 8px.